src/Toast.js

```diff
@@ -79,7 +79,7 @@ const Toast = ({
       };
     }
     return () => null;
-  }, [autohide, show]);
+  }, [autohide, delay, onClose, show]);
```

Comment:
Was a ESLint warning

Reply:
hmm, this was deliberate though. we didn't want to restart the timer in the event that `delay` changed. though `onClose` should probably take the latest value via a ref